docker/Dockerfile.fortran-nvhpc

```
@@ -6,7 +6,7 @@
 #
 FROM nvcr.io/nvidia/nvhpc:23.7-devel-cuda12.2-ubuntu22.04
 
-ARG MUSICA_GIT_TAG=main
+ARG MUSICA_GIT_TAG=801807d
```

```suggestion
ARG MUSICA_GIT_TAG=main
```

In the future, you can use `docker build -t musica --build-arg="MUSICA_GIT_TAG=801807d"`

---

That's good to know